### ISSUE TITLE
(ocmod) Path cannot start with brace  e.g. {admin,catalog}/  not allowed

### ISSUE THREAD

**Issue by dtjones** [opencart/opencart#2561]
I've done a quick but ugly fix to admin/controller/extensions/modification.php. 
The $file variable is overwritten during the loop so ideally that would need renaming throughout.  

        $files = $dom->getElementsByTagName('modification')->item(0)->getElementsByTagName('file');

        foreach ($files as $file) {
            $operations = $file->getElementsByTagName('operation');  
                                               
            $file_tag = $file;
            $parts = explode('/', $file->getAttribute('path') ,2);
            foreach (explode(',',str_replace(array('{','}'),'', $parts[0])) as $key => $dir) {
                $file = $file_tag;
                $file->setAttribute('path', $dir . '/' . $parts[1]);

                /* current code */

            }
        }
        // Log
        $log[] = '----------------------------------------------------------------';

**Comment by rmullaney77** [comment]
can't you just use `path="./{admin,catalog}/"`?

**Comment by dtjones** [comment]
the current code makes sure the path starts with 'system','catalog', or 'admin e.g

           if (substr($file->getAttribute('path'), 0, 7) == 'catalog')

**Comment by rmullaney77** [comment]
ah ok, sorry

**Comment by dtjones** [comment]
Quick question: 
Will this now work?

     <file path="{admin,catalog}/language/english/default.php"> 
Quick answer:
Yes or No ?


I don't want to waste anyone's time but I have looked at the commits and I cannot see how the issue is fixed. I don't need an explanation I just wont to make sure the correct issue had been closed as the first commit doesn't seem to relate to this issue at all. 
@rmullaney77 @danielkerr
